### PR TITLE
python3 compatibility issue

### PR DESCRIPTION
**Pull request by ryanwcyin**
Fixed python3 compatibility issue #4780

### CODE REVIEW

File: research/object_detection/model_lib.py [hunk 1]
```diff
@@ -410,8 +411,9 @@ def tpu_scaffold():
             eval_dict)
 
       # Eval metrics on a single example.
+      category_index_values = category_index.values() if sys.version_info[0] < 3 elses list(category_index.values())
```
Comment (rob-white) [inline research/object_detection/model_lib.py]: Currently throws a syntax error:

Change `elses` to `else`

**Comment by ryanwcyin** [comment]
@rob-white Thanks for the reminder! the typo is fixed:)

**Comment by cclauss** [comment]
Two other Python 3 issues in object_detection...

[flake8](http://flake8.pycqa.org) testing of https://github.com/tensorflow/models on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./research/object_detection/dataset_tools/oid_hierarchical_labels_expansion.py:157:55: E999 SyntaxError: invalid syntax
    print '--annotation_type expected value is 1 or 2.'
                                                      ^
./research/object_detection/utils/object_detection_evaluation.py:307:27: F821 undefined name 'unicode'
          category_name = unicode(category_name, 'utf-8')
                          ^
1     E999 SyntaxError: invalid syntax
1     F821 undefined name 'unicode'
89
```

**Comment by cclauss** [comment]
Two other Python 3 issues in object_detection...

[flake8](http://flake8.pycqa.org) testing of https://github.com/tensorflow/models on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./research/object_detection/dataset_tools/oid_hierarchical_labels_expansion.py:157:55: E999 SyntaxError: invalid syntax
    print '--annotation_type expected value is 1 or 2.'
                                                      ^
./research/object_detection/utils/object_detection_evaluation.py:307:27: F821 undefined name 'unicode'
          category_name = unicode(category_name, 'utf-8')
                          ^
1     E999 SyntaxError: invalid syntax
1     F821 undefined name 'unicode'
89
```

**Comment by cclauss** [comment]
Two other Python 3 issues in object_detection...

[flake8](http://flake8.pycqa.org) testing of https://github.com/tensorflow/models on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./research/object_detection/dataset_tools/oid_hierarchical_labels_expansion.py:157:55: E999 SyntaxError: invalid syntax
    print '--annotation_type expected value is 1 or 2.'
                                                      ^
./research/object_detection/utils/object_detection_evaluation.py:307:27: F821 undefined name 'unicode'
          category_name = unicode(category_name, 'utf-8')
                          ^
1     E999 SyntaxError: invalid syntax
1     F821 undefined name 'unicode'
89
```

**Comment by cclauss** [comment]
Two other Python 3 issues in object_detection...

[flake8](http://flake8.pycqa.org) testing of https://github.com/tensorflow/models on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./research/object_detection/dataset_tools/oid_hierarchical_labels_expansion.py:157:55: E999 SyntaxError: invalid syntax
    print '--annotation_type expected value is 1 or 2.'
                                                      ^
./research/object_detection/utils/object_detection_evaluation.py:307:27: F821 undefined name 'unicode'
          category_name = unicode(category_name, 'utf-8')
                          ^
1     E999 SyntaxError: invalid syntax
1     F821 undefined name 'unicode'
89
```

**Comment by cclauss** [comment]
Two other Python 3 issues in object_detection...

[flake8](http://flake8.pycqa.org) testing of https://github.com/tensorflow/models on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./research/object_detection/dataset_tools/oid_hierarchical_labels_expansion.py:157:55: E999 SyntaxError: invalid syntax
    print '--annotation_type expected value is 1 or 2.'
                                                      ^
./research/object_detection/utils/object_detection_evaluation.py:307:27: F821 undefined name 'unicode'
          category_name = unicode(category_name, 'utf-8')
                          ^
1     E999 SyntaxError: invalid syntax
1     F821 undefined name 'unicode'
89
```

**Comment by cclauss** [comment]
Two other Python 3 issues in object_detection...

[flake8](http://flake8.pycqa.org) testing of https://github.com/tensorflow/models on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./research/object_detection/dataset_tools/oid_hierarchical_labels_expansion.py:157:55: E999 SyntaxError: invalid syntax
    print '--annotation_type expected value is 1 or 2.'
                                                      ^
./research/object_detection/utils/object_detection_evaluation.py:307:27: F821 undefined name 'unicode'
          category_name = unicode(category_name, 'utf-8')
                          ^
1     E999 SyntaxError: invalid syntax
1     F821 undefined name 'unicode'
89
```

**Comment by jaeyounkim** [comment]
Fixed

**Pull request closed without merging**